app/uk/gov/hmrc/trusts/controllers/TrustVariationsController.scala

```diff
@@ -19,32 +19,27 @@ package uk.gov.hmrc.trusts.controllers
 import javax.inject.Inject
 import play.api.Logger
 import play.api.libs.json.Json
-import uk.gov.hmrc.trusts.config.AppConfig
 import uk.gov.hmrc.trusts.controllers.actions.IdentifierAction
 import uk.gov.hmrc.trusts.exceptions._
 import uk.gov.hmrc.trusts.models.auditing.TrustAuditing
-import uk.gov.hmrc.trusts.models.variation.Variation
-import uk.gov.hmrc.trusts.services.{AuditService, DesService, ValidationService}
+import uk.gov.hmrc.trusts.models.variation.TrustVariation
+import uk.gov.hmrc.trusts.services.{AuditService, DesService}
 import uk.gov.hmrc.trusts.utils.ErrorResponses._
 import uk.gov.hmrc.trusts.utils.ValidationUtil
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class VariationsController @Inject()(
+class TrustVariationsController @Inject()(
                                       identify: IdentifierAction,
-                                      config: AppConfig,
                                       desService: DesService,
-                                      auditService: AuditService,
-                                      validationService: ValidationService
+                                      auditService: AuditService
                                     ) extends TrustsBaseController with ValidationUtil {
 
-  def variation() = identify.async(parse.json) {
+  def trustVariation() = identify.async(parse.json) {
     implicit request =>
 
-      val payload = request.body.toString
-
-      validationService.get(config.variationsApiSchema).validate[Variation](payload).fold(
+      request.body.validate[TrustVariation].fold(
```

Comment:
You have removed the `validationService` and the call to `validationService.get(config.variationsApiSchema).validate[Variation](payload).fold()` due to the way the branch was created off the branch and merged back in.
The commits were squashed and this has resulted in the bad merge.

Reply:
Added validation service to tests and implementations